main.go

```diff
@@ -83,6 +101,40 @@ func LoggerSetup(info io.Writer) {
 	logger = log.New(info, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile)
 }
 
+func checkAPI(baseURL string, token string, namespace string, project string) (string, error) {
+	gitlab := "gitlab"
+	github := "github"
```

Comment:
Could be simplified to just use "gitlab"/"github" within the respective cases. Would be closer to how you do this in other functions.

Reply:
Thanks, I've made these changes for simplicity/better consistency.